### ISSUE TITLE
[Todoist] Create Apple Reminders With Todoist Tasks

### ISSUE THREAD

**Issue by geraldaddey** [raycast/extensions#16068]
### Extension

https://www.raycast.com/doist/todoist

### Description

Currently, the Todoist extension allows users to interact with their Todoist tasks but lacks the ability to create reminders in Apple Reminders using task details. Since only Pro users can create Todoist reminders, this feature will enable all users, including free-tier users, to leverage Apple Reminders for notifications across all Apple devices.

### Who will benefit from this feature?

Free-tier Todoist users who lack access to Todoist reminders.
Users who want seamless integration with Apple's ecosystem for device-wide notifications.
Individuals managing tasks across Todoist and Apple Reminders who need cross-platform functionality

**Comment by thomaslombart** [comment]
I'm not sure I got what you mean. The use-case is about creating a reminder in the Apple Reminders app from Todoist? 🤔

**Comment by geraldaddey** [comment]
Yes indeed. Right from a selected Todoist task.  This would benefit free tier users who can easily create a reminder from a todoist task with a command. Reminders is a reserved feature for pro users but this certainly serves as an alternative to allow users to receive notifications from their tasks as reminders on all apple devices.

**Comment by thomaslombart** [comment]
But didn't Todoist make Reminders free for everyone a while ago? Either way, while I understand the use case, I don't see that happening with the extension since it would open the door to all kinds of third-party integrations between all extensions 😅

That being said, we have something in the works that could partially solve this issue, so stay tuned!

**Comment by thomaslombart** [comment]
Hey there, we've released AI extensions that are already supported in Todoist and Apple Reminders. If you have Raycast Pro, it should make it easier for you to add tasks in both apps!
